src/Client.php

```diff
@@ -23,19 +30,30 @@ class Client
      */
     public function __construct($accessKey)
     {
+        $this->initializeEnv();
+
         $this->accessKey = $accessKey;
         $this->client = new GuzzleClient([
-            'base_uri' => 'http://127.0.0.1:8000/api/',
+            'base_uri' => $_ENV['BASE_URL'],
```

Comment:
A third party package should always set a default url for there API lets use the prod url

Reply:
`$_ENV['BASE_URL'] ?? 'url'`